packages/sdk/src/types/runtime/actor.ts

```diff
@@ -345,47 +304,73 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * Adds a collider of the given type and parameters on the actor.
 	 * @param colliderType Type of the collider to enable.
 	 * @param isTrigger Whether the collider is a trigger volume or not.
+	 * @param radius The radius of the collider.
 	 * @param center The center of the collider, or default of the object if none is provided.
-	 * @param radius The radius of the collider, or default bounding if non is provided.
 	 */
 	// * @param collisionLayer The layer that the collider operates in.
 	public setCollider(
 		colliderType: 'sphere',
 		// collisionLayer: CollisionLayer,
 		isTrigger: boolean,
-		center?: Vector3Like,
-		radius?: number): void;
+		radius: number,
+		center?: Vector3Like
+	): void;
 
 	/**
 	 * Adds a collider of the given type and parameters on the actor.
 	 * @param colliderType Type of the collider to enable.
 	 * @param isTrigger Whether the collider is a trigger volume or not.
+	 * @param size The dimensions of the collider.
 	 * @param center The center of the collider, or default of the object if none is provided.
-	 * @param size
 	 */
 	public setCollider(
 		colliderType: 'box',
 		// collisionLayer: CollisionLayer,
 		isTrigger: boolean,
-		center?: Vector3Like,
-		size?: Vector3Like): void;
+		size: Vector3Like,
+		center?: Vector3Like
+	): void;
+
+	/**
+	 * Adds a collider of the give type and parameters on the actor.
+	 * @param colliderType Type of the collider to enable.
+	 * @param isTrigger Whether the collider is a trigger volume or not.
+	 * @param size The dimensions of the collider, with the largest component of the vector
+	 * being the primary axis and height of the capsule, and the smallest the radius.
+	 * @param center The center of the collider, or default of the object if none is provided.
+	 */
+	public setCollider(
+		colliderType: 'capsule',
+		isTrigger: boolean,
+		size: Vector3Like,

```

Comment:
>size: Vector3Like, [](start = 2, length = 18)

We discussed this that the Unity client provides a reasonable default for this when specified, and that optional should be allowed for the size related properties here.  These should still be optional.

Reply:
Changed back, and documented default.

---
In reply to: [315736742](https://github.com/microsoft/mixed-reality-extension-sdk/pull/415#discussion_r315736742) [](ancestors = 315736742)